pos/src/main/java/edu/illinois/cs/cogcomp/pos/POSAnnotator.java

```diff
@@ -123,4 +126,23 @@ public void addView(TextAnnotation record) throws AnnotatorException {
     public String getAnnotatorName() {
         return NAME;
     }
+
+    /**
+     * Return possible tag values that the POSAnnotator can produce.
+     * The tag information for POS is found here:
+     * http://cogcomp.cs.illinois.edu/page/demo_view/POS
+     *
+     * @return the set of string representing the tag values
+     */
+    @Override
+    public Set<String> getTagValues() {
+        if (!isInitialized()) {
+            doInitialize();
+        }
+        String tagValues[] = {"#", "$", "''", "``", "'", "`", ",", ".", ":", "-LRB-", "-RRB-", "CC", "CD",
+                "DT", "EX", "FW", "IN", "JJ", "JJR", "JJS", "LS", "MD", "NN", "NNS", "NNP", "NNPS", "PDT",
+                "POS", "PRP", "PP$", "RB", "RBR", "RBS", "RP", "SYM", "TO", "UH", "VB", "VBD", "VBG", "VBN",
+                "VBP", "VBZ", "WDT", "WP", "WP$", "WRB"};
```

Comment:
I think it would be great if we collect this list automatically, rather than listing them all. 
I would try looking into the `tagger. getLabeler(). allowableValues()`:

https://github.com/IllinoisCogComp/lbjava/blob/master/lbjava/src/main/java/edu/illinois/cs/cogcomp/lbjava/classify/Classifier.java#L137